### PR TITLE
Add golines

### PR DESCRIPTION
**Pull request by fredrikaverpil**
https://github.com/segmentio/golines

### CODE REVIEW

File: tools/sggolines/tools.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+)
+
+// Command returns an [*exec.Cmd] for golines.
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
+	_, err := sgtool.GoInstall(ctx, "github.com/segmentio/golines", version)
```
Comment (odsod) [inline tools/sggolines/tools.go:23]: IMO since golines publishes pre-built artifacts let's use those:
https://github.com/segmentio/golines/releases

Comment (odsod) [reply tools/sggolines/tools.go:23]: (faster to install, especially on GitHub Actions when it's fetched from within the same server rack :smile:)

**Review by odsod — COMMENTED** [review]
Great initiative!

IMO we could add `GoFormat` as a default target to the Sage template, but then we should also run GolangCI-Lint in fix mode (which should fix imports, protogetters, and whatnot)

File: tools/sggolines/tools.go (hunk 1 above)
Comment (fredrikaverpil) [reply tools/sggolines/tools.go:23]: Fixed... but I believe github is having issues updating the PR: https://githubstatus.com

Comment (fredrikaverpil) [reply tools/sggolines/tools.go]: Ok, now it looks like the PR got updated. 😆

**Review by odsod — APPROVED** [review]

**Pull request merged**